openlibrary/i18n/__init__.py

```diff
@@ -15,7 +14,7 @@
 
 def _compile_translation(po, mo):
     try:
-        catalog = read_po(open(po))
+        catalog = read_po(open(po, 'rb'))
```

Comment:
@tfmorris More out of curiosity, but is there a quick explanation of why this needed to change from `r` to `rb`? Just wondering in case it becomes relevant for Python 3 upgrades, or Unicode handling subtleties in future. I feel like I should understand, but I have no objection to the change.

Reply:
This is actually a Python 3 change. The related changes are all in commit f69aa934ddda34eb153780733e59a9ad5b33026e